.gitignore

```diff
@@ -84,3 +84,11 @@ src/
 website/mfrtemp/
 docs/_build
 website/static/img/badges
+
+# Emacs temporary/lock files
```

Comment:
These should go in your global gitignore: https://help.github.com/articles/ignoring-files/#create-a-global-gitignore